src/styles/blocks/button.scss

```
@@ -0,0 +1,30 @@
+.card {
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border: 1px solid $button-color;
+    border-radius: 5px;
+    background-color: $button-color;
+
+    width: 166px;
+    height: 40px;
+
+    &-text {
+      font-weight: 700;
+      font-size: 14px;
+      line-height: 16px;
+      color: #fff;
```

don't forget to create variables for all colors
```suggestion
      color: $c-white;
```